npm/examples.js

```diff
@@ -7,7 +7,7 @@ const examplesDir = 'examples';
 log.task('examples');
 
 // Load asciidoctor.js and local asciidoctor-reveal.js
-const asciidoctor = require('asciidoctor.js')();
+const asciidoctor = require('@asciidoctor/core')();
 const asciidoctorRevealjs = require('../build/asciidoctor-reveal.js');
```

Comment:
This API change leads me to believe the README also needs updating?

Reply:
Yes :+1: